### ISSUE TITLE
Filum terminale lipoma

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7713]
**Preferred term label:**
Filum terminale lipoma

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
618351 
PMID: 30715179  1/3

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000184
- label: Filum terminale lipoma
- parents: [Spinal cord tumor (HP:0010302)](https://hpo.jax.org/app/browse/term/HP:0010302)
- pmid: PMID:24628511
- def: Filum terminale lipomas (FTLs) are a type of lumbosacral lipoma in which the fat is entirely within the filum terminale and separate from the conus medullaris.
- comment: Filum terminale lipomas (FTL) are occasionally associated with tethered cord syndrome, and some authors have advocated for prophylactic sectioning of the filum terminale when an FTL is identified. With the increasing use of MRI, FTLs have been identified more frequently in asymptomatic individuals.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7713
- synonyms:
